resources/lang/fr/buttons.php

```diff
@@ -18,14 +18,14 @@
             'users' => [
                 'activate'           => 'Activer',
                 'change_password'    => 'Changer de mot de passe',
-                'clear_session'         => 'Clear Session',
-                'confirm'             => 'Confirm',
+                'clear_session'       => 'Effacer la session',
```

Comment:
Don't worry about formatting, StyleCI has an auto hook that formats when I pull to dev.